gapis/service/path/validate.go

```diff
@@ -171,22 +171,22 @@ func (n *DeviceTraceConfiguration) Validate() error {
 
 // Validate checks the path is valid.
 func (n *Events) Validate() error {
-	return checkNotNilAndValidate(n, protoutil.OneOf(n.Capture), "capture")
+	return checkNotNilAndValidate(n, n.Capture, "capture")
 }
 
 // Validate checks the path is valid.
 func (n *FramebufferObservation) Validate() error {
-	return checkNotNilAndValidate(n, protoutil.OneOf(n.Command), "command")
+	return checkNotNilAndValidate(n, n.Command, "command")
 }
 
 // Validate checks the path is valid.
 func (n *FramebufferAttachments) Validate() error {
-	return checkNotNilAndValidate(n, protoutil.OneOf(n.After), "command")
+	return checkNotNilAndValidate(n, n.After, "command")
```

Comment:
Not directly related to the purpose of this PR, but should we also fix a nit btw, by changing the third parameter to "after"? same for the next changed line.

Reply:
Done.